Revise BLISS results section (#1440)
ihmeuw/vivarium_research: docs/source/model_design/general_reference_material/bliss/index.rst

  Despite prevailing belief that sex affects gender but not vice versa, there is
  strong evidence for causal influence in both directions. [Ritz_and_Greaves_2022]_ [Springer_2012]_
  Even when we do think that a sex-related factor is most causally related to our outcome,
  we should still mention that some of its impact could be mediated through
  gender factors and that it is difficult to disentangle these.
* Highlight data limitations around sex and gender measures.
  We don't want our inclusive language to give the impression that we are able to make accurate
  conclusions about sexual and gender minority populations when the data prevent us from doing this.
  Furthermore, drawing attention to the problems with how almost every data source currently
  captures sex and/or gender raises awareness that changes need to be made in primary data collection.

4.2 Strategy
------------

We often need to be very concise, and can't surface all the complexity
of sex and gender every time we reference them. So, we take the following
approach:

* Somewhere in the paper/report, we explain in a fair amount of detail the

>   sex and gender measures we used, why we used them that way, etc.

Reviewer: For example... (include example from VEHSS paper?)

4.3 Detailed description
------------------------

The detailed description of our methods regarding sex and gender can be centralized
in one place, or spread throughout the paper or report.
Wherever it is, it should include most of the considerations described in earlier
sections of this guide: defining the variables of interest for our
research question and noting the difficulty of disentangling factors from each other,
giving details about the sex- and gender-related measures in the data sources used,
and (usually) calling for better data to be collected.

Here are a few suggestions for the detailed description:

* If we can, cite some of the research that is cited in this guide!
  This gives curious readers a starting point to learn more,
  and gives credit to the researchers whose ideas we are building on.
  See the Recommended Resources section below.
* If we are not able to present quantitative results for sexual and
  gender minority populations, but have reason to believe that these
  populations are especially impacted by health issues (this will be
  true for almost any health condition in the US), we can cite some

>   tangentially related quantitative research or qualitative research

Reviewer: ```suggestion
  relevant quantitative research or qualitative research
```

4.0 Results
+++++++++++

4.1 Goals when Reporting Results
--------------------------------

When we present our findings, we have two primary goals:

* Accurately convey the variables of interest, and what the causal structures might be.
  We should strive to be as precise as possible, and not rely on "common sense"
  to indicate which gender- and sex-related factors are related to a health outcome.
  Despite prevailing belief that sex affects gender but not vice versa, there is
  strong evidence for causal influence in both directions. [Ritz_and_Greaves_2022]_ [Springer_2012]_
  Even when we do think that a sex-related factor is most causally related to our outcome,
  we should still mention that some of its impact could be mediated through
  gender factors and that it is difficult to disentangle these.
* Highlight data limitations around sex and gender measures.
  We don't want our inclusive language to give the impression that we are able to make accurate

>   conclusions about sexual and gender minority populations when the data prevent us from doing this.

Reviewer 2: ```suggestion
  conclusions about SGM populations when the data prevent us from doing this.
```
we defined this acronym in the glossary, so could use shorthand version? especially if we use this acronym at other points in the guide

>   Furthermore, drawing attention to the problems with how almost every data source currently

Reviewer 2: I know this isn't new, but just wanted to emphasize how important this point is!! The Rioux et al (2022) paper also made a similar point, that inclusive language/reporting can work in tandem (and become less difficult/complicated) with improved data collection methods. 

_Accordingly, since the gender of pregnant participants is currently rarely reported or measured, our recommendations favour gender-inclusive language as more accurate language. In the future however, the field should move towards measuring and reporting participants’ gender modality/identity as any other demographic measures. As this becomes the norm, cis women-related terms and language inclusive of gender diversity will be able to better coexist by qualifying sampled participants and studied pregnant populations precisely and appropriately, which will avoid erasure of both cisgender women and gender-diverse pregnant people._

I wonder if we should add something similar to this in our Guiding Principles, like the 'This is Work' section.

Author: It would be great to cite this Rioux et al paper! I think it could support several different parts of the guide 😃

Most journal editors also consider tables and figures to stand alone in this way (i.e. they cannot use
acronyms from the main text), so we follow that convention with our sex and gender terminology.

The following sections give more concrete recommendations for each part of results reporting.

4.3 Detailed description
------------------------

The detailed description of our methods regarding sex and gender can be centralized
in one place, or spread throughout the paper or report.
Wherever it is, it should include most of the considerations described in earlier
sections of this guide: defining the variables of interest for our
research question and noting the difficulty of disentangling factors from each other,
giving details about the sex- and gender-related measures in the data sources used,
and (usually) calling for better data to be collected.

Here are a few suggestions for the detailed description:

* If we can, cite some of the research that is cited in this guide!
  This gives curious readers a starting point to learn more,
  and gives credit to the researchers whose ideas we are building on.
  See the Recommended Resources section below.

> * If we are not able to present quantitative results for sexual and

Reviewer 2: Another example where we could switch to acronym if desired!

  Reference 'measurement error' section again here.

4.4 Result indicator definition
-------------------------------

> In *each* stand-alone document (paper, table, figure, etc) where we present results

Reviewer: does this concept of stand-alone document make sense to everyone?  I'm interpreting it to mean in the main paper, in each table, and in each figure, since any table or figure might be referred to in isolation by a busy scientist.

Reviewer 3: I think this makes sense to me.

Author: Hm I might take a crack at spelling this out more explicitly regardless.

that relate to sex and/or gender,
we should define our result indicator (or indicators, though we
usually won't be able to report more than one).

How an indicator should be defined depends on the data that informed it.
In general, we think of an indicator as the "least common denominator"
of the data that went into it.

Put another way, when the sex/gender-related factors we are reporting on are unclear - 
for example when we have assumed equivalence between different measures across 
data sources - we should use terms that reflect this uncertainty.

To understand this, we can look at our figure of hypothetical data sources
from above. There is now an additional row with how we would include
this data in our analysis. You can see the first two data sources would both be
undifferentiated sex/gender. This is due to ambiguity between sex and gender in
the question asked.

The latter two data sources more clearly reflect sex, although one is sex assigned

> at birth and the other is sex identity. Therefore if you included only data sources

Reviewer: ```suggestion
at birth and the other is gender identity. Therefore if you included only data sources
```
?

Reviewer 3: No, I set up this figure to ask explicitly about what is your sex, partially to set up a section saying that these data sources both clearly reference sex.

Author: Yes, agree that it should be "sex" and not "gender" (note that I didn't edit this in this PR, it is just moved around). On a cursory look at our cited survey methodology sources, I actually can't find a reference for this meaning of the term "sex identity" -- perhaps something like "self-reported current sex" could be less ambiguous. See for example [the "Allowable alternative question module" for sex from the Australian Bureau of Statistics.](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#sex)
